core/new-gui/src/app/workspace/component/navigation/navigation.component.ts

```diff
@@ -84,7 +86,9 @@ export class NavigationComponent {
     public userService: UserService,
     private workflowCacheService: WorkflowCacheService,
     private datePipe: DatePipe,
-    public workflowResultExportService: WorkflowResultExportService
+    public workflowResultExportService: WorkflowResultExportService,
+    private modalService: NgbModal,
```

Comment:
I thought we are not using bootstrap modal?

Reply:
fixed, only nzModal now.